cli/version_check.go

```diff
@@ -33,6 +34,7 @@ func PopulateTerraformVersion(terragruntOptions *options.TerragruntOptions) erro
 	}
 
 	terragruntOptions.TerraformVersion = terraformVersion
+	util.Debugf(util.CreateLogger(""), "%s", terraformVersion)
```

Comment:
Could you use `terragruntOptions.Logger` instead of creating a new one?

Also, adding a label like "Terraform version: %s" will make the log output easier to understand.

Reply:
> Could you use `terragruntOptions.Logger` instead of creating a new one?
> 
> Also, adding a label like `"Terraform version: %s"` will make the log output easier to understand.

Ok, changes made.